src/Nancy/HttpLink.cs

```diff
@@ -0,0 +1,262 @@
+namespace Nancy
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using Nancy.Responses.Negotiation;
+
+    /// <summary>
+    /// Represents one of possibly many RFC 5899 HTTP Links contained in a <see cref="HttpLinkBuilder"/>.
+    /// </summary>
+    public class HttpLink : Dictionary<string, object>, IEquatable<HttpLink>
```

Comment:
It's an easy (but perhaps hard to unintuitive, now that you mention it) way to expose the link's parameters. Perhaps I should move it into a `Parameters` property?